### DISCUSSION TITLE
Configure Prettier and ESLint with Angular 🎨 - justangular.com

### DISCUSSION THREAD

**Discussion by eneajaho** [eneajaho/just-angular#4] [General]
https://justangular.com/blog/configure-prettier-and-eslint-with-angular

**Comment by chrispinzaru** [comment]
Great article Enea! 
Just recently I've been as always installing all these tools, and found myself having this Article in an open browser tab :)
It'd be much appreciated if you could spare a few words on Git & pre-commit/pre-push hooks (for example using the beloved Husky) as well as tools like Commitlint (for Conventional Commits lovers <3) that would force the whole team format & lint all the changes upon the attempt to commit something.
Adding smth like "ng test" to the pre-push hook might be a good idea as well (no need IMO if you have a proper CI/CD pipeline configured and you run tests for each open MR/PR) 
Thank you! :)

**Comment by PrSalvator** [comment]
thank you for this amazing article i want to learn angulyay and evneloupe very much!!!!

**Comment by ben-barbier** [comment]
Thanks for the article!
What’s your take on [prettier-plugin-organize-imports](https://www.npmjs.com/package/prettier-plugin-organize-imports)?
It seems to remove unused imports and sort them at the same time 🙂
